db/migrate/20160317160230_move_user_roles_to_entitlements.rb

```diff
@@ -0,0 +1,16 @@
+class MoveUserRolesToEntitlements < ActiveRecord::Migration[5.0]
+  class MiqGroup < ActiveRecord::Base; end
+  class Entitlement < ActiveRecord::Base; end
+
+  def up
+    MiqGroup.find_each do |group|
+      Entitlement.create!(:miq_group_id => group.id, :miq_user_role_id => group.miq_user_role_id)
+    end
+  end
+
+  def down
+    MiqGroup.find_each do |group|
+      Entitlement.find_by(:miq_group_id => group.id).try(:delete)
+    end
```

Comment:
@jrafanie Heh, forgot about this one, which I'll do unless you object: `Entitlement.delete_all(:miq_group_id => group.id)`